data/en/numberformat.json

```diff
@@ -43,6 +43,11 @@
 			"code":"num1 = NumberFormat( 5, \"+\" );\r\nnum2 = NumberFormat( -5, \"+\" );\r\n    \r\nwriteOutput(num1&'/'&num2);",
 			"result":"+5/-5",
 			"runnable":true
+		},{
+			"title":"Formats to a dollar format (US)",
+			"description":"",
```

Comment:
Would it be a good idea to relate this example to [`dollarFormat()`](http://cfdocs.org/dollarformat) and  [`lsCurrencyFormat()`](http://cfdocs.org/lscurrencyformat)?